contrib/seekable_format/zstd_seekable.h

```diff
@@ -29,6 +29,7 @@ extern "C" {
 
 typedef struct ZSTD_seekable_CStream_s ZSTD_seekable_CStream;
 typedef struct ZSTD_seekable_s ZSTD_seekable;
+typedef struct ZSTD_seekTable_s ZSTD_seekTable;
```

Comment:
just a note : 
the name `ZSTD_seekTable` looks correct,
I just can't help but find it close to `ZSTD_seekable`, 
which could become a source of confusion.

I don't have a better idea in mind though.